frontend/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -142,11 +142,13 @@ export class DeckGlJsonChart extends PureComponent<PropsWithHeight, State> {
     const { element, width, height, theme } = props
     const json = JSON.parse(element.json)
 
-    // Use either the Mapbox light or dark style based on Streamlit's theme
+    // If unset, use either the Mapbox light or dark style based on Streamlit's theme
     // For Mapbox styles, see https://docs.mapbox.com/api/maps/styles/#mapbox-styles
-    const hasLightBg = getLuminance(theme.colors.bgColor) > 0.5
-    const mapTheme = hasLightBg ? "light" : "dark"
-    json.mapStyle = `mapbox://styles/mapbox/${mapTheme}-v10`
+    if (typeof json.mapStyle === "undefined") {
+      const hasLightBg = getLuminance(theme.colors.bgColor) > 0.5
+      const mapTheme = hasLightBg ? "light" : "dark"
+      json.mapStyle = `mapbox://styles/mapbox/${mapTheme}-v9`
```

Comment:
style version is back to `v9`. Is that because of the faster rendering?

Reply:
No, I had no reason to  make the switch to `v10`, so I reverted to the status quo.  I have no strong opinions on this though..

Reply:
ok, yes I think we can leave it out of this PR if we don't know any specific benefits. At least it would prevent us from pushing new snapshots.